storage/api/Storage.Samples.Tests/ChangeFileStorageClassTest.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.Storage.V1;
+using Xunit;
+
+[Collection(nameof(BucketFixture))]
+public class ChangeFileStorageClassTest
+{
+    private readonly BucketFixture _bucketFixture;
+
+    public ChangeFileStorageClassTest(BucketFixture bucketFixture)
+    {
+        _bucketFixture = bucketFixture;
+    }
+
+    [Fact]
+    public void ChangeFileStorageClass()
+    {
+        ChangeFileStorageClass changeClass = new ChangeFileStorageClass();
```

Comment:
```suggestion
        ChangeFileStorageClass changeFileStorageClass = new ChangeFileStorageClass();
```

make this changes in all the tests.